plugins/USBPrinting/USBPrinterOutputController.py

```diff
@@ -31,6 +31,17 @@ def homeHead(self, printer):
     def homeBed(self, printer):
         self._output_device.sendCommand("G28 Z")
 
+    def setJobState(self, job: "PrintJobOutputModel", state: str):
+        if state == "pause":
```

Comment:
Would it be possible to use an enum for possible states instead of using hardcoded string everywhere this is used?